app/form_objects/support/urns_and_allocations_form.rb

```diff
@@ -0,0 +1,39 @@
+class Support::UrnsAndAllocationsForm
+  include ActiveModel::Model
+
+  attr_accessor :urns_and_allocations, :parsed_list
+
+  validates :urns_and_allocations, presence: { message: 'Enter the school URNs and new allocations, one pair per line' }
+
+  def initialize(args = {})
+    @urns_and_allocations = args[:urns_and_allocations]
+  end
+
+  def parse_urns_and_allocations!
+    return [] if urns_and_allocations.empty?
+
+    @parsed_list = urns_and_allocations.split("\r\n")
+                                        .map(&:strip)
+                                        .reject(&:blank?)
+                                        .map { |line| line.split(/[\s+,]/) }
```

Comment:
The following might be a bit more robust?
```
[37] pry(main)> "1      3".split(/\s*,\s*|\s*/)
=> ["1", "3"]
[38] pry(main)> "1  ,    3".split(/\s*,\s*|\s*/)
=> ["1", "3"]
```

Reply:
good call - I've gone with the more concise but equally effective adjustment: 
`/[\s+,]/` -> `/[\s,]+/`
and added a feature spec